src/app/modules/submission/email-submission/email-submission.receiver.ts

```diff
@@ -0,0 +1,130 @@
+import Busboy from 'busboy'
+import { IncomingHttpHeaders } from 'http'
+import { err, ok, Result, ResultAsync } from 'neverthrow'
+
+import { createLoggerWithLabel } from '../../../../config/logger'
+import { MB } from '../../../constants/filesize'
+
+import {
+  InitialiseMultipartReceiverError,
+  MultipartContentLimitError,
+  MultipartContentParsingError,
+  MultipartError,
+} from './email-submission.errors'
+import { IAttachmentInfo, ParsedMultipartForm } from './email-submission.types'
+import {
+  addAttachmentToResponses,
+  handleDuplicatesInAttachments,
+} from './email-submission.util'
+
+const logger = createLoggerWithLabel(module)
+
+export const createMultipartReceiver = (
+  headers: IncomingHttpHeaders,
+): Result<busboy.Busboy, InitialiseMultipartReceiverError> => {
```

Comment:
Does `busboy.Busboy` work? Interesting. Can we just use `Busboy` since it's already imported?

Reply:
I tried, didn't work! `busboy` is the namespace